apps/sms/views/conversation/js/conversation.js

```diff
@@ -596,7 +597,11 @@ var ConversationView = {
       var emailThread = Settings.supportEmailRecipient &&
         this.activeThread.participants.some(Utils.isEmailAddress);
 
-      Compose.setupLock({ forceType: () => emailThread ? 'mms' : null });
+      Compose.setupLock({
```

Comment:
I simply reuse the setupLock to make the changes as less as possible, but it might be difficult if we lowDiskSpace is returned as async.